charts/kubernikus-system/charts/prometheus/templates/_prometheus.yaml.tpl

```diff
@@ -202,8 +202,46 @@ scrape_configs:
     action: replace
     replacement: ${1}
 
-# Static Targets 
-#
+- job_name: 'kube-system/kubelet'
+  kubernetes_sd_configs:
+  - role: node
+  relabel_configs:
+  - action: labelmap
+    regex: __meta_kubernetes_node_label_(.+)
+  - target_label: component
+    replacement: kubelet
+  - action: replace
+    source_labels: [__meta_kubernetes_node_name]
+    target_label: instance
+  - source_labels: [__address__]
+    action: replace
+    target_label: __address__
+    regex: ([^:;]+):(\d+)
+    replacement: ${1}:10255
+  - source_labels: [__scheme__]
+    action: replace
+    target_label: __scheme__
+    regex: https
+    replacement: http
```

Comment:
this is still necessary to avoid ```Get https://198.18.0.22:10250/metrics: x509: cannot validate certificate for 198.18.0.22 because it doesn't contain any IP SANs```

Reply:
Ok, we can't add the whole node subnet as IPs to the certificates. So, here we need to grab the unsecured port or skip host validation